fdbclient/ServerKnobs.cpp

```diff
@@ -577,6 +577,10 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( INITIAL_DURABILITY_LAG_MULTIPLIER,                    1.02 );
 	init( DURABILITY_LAG_REDUCTION_RATE,                      0.9999 );
 	init( DURABILITY_LAG_INCREASE_RATE,                        1.001 );
+	init( INITIAL_READ_QUEUE_MULTIPLIER,                        1.02 );
+	init( READ_QUEUE_REDUCTION_RATE,                          0.9999 );
+	init( READ_QUEUE_INCREASE_RATE,                            1.001 );
+	init( MAX_READ_QUEUE,                                  200000000 ); // This is set sufficiently high to be effectively disabled by default.
```

Comment:
100k is probably a very large number already.

Reply:
@sfc-gh-satherton specifically told me 100k was not large enough.

Reply:
My reasoning was that >100k small reads per second is attainable so it's only 1s worth of requests in that case.

This can be buggified to something small in simulation to make sure it doesn't cause the cluster to get stuck or anything else really bad to happen.

Reply:
Added buggify